flash/core/classification.py

```diff
@@ -21,12 +21,15 @@
 from flash.core.data.data_source import DefaultDataKeys, LabelsState
 from flash.core.data.process import Serializer
 from flash.core.model import Task
-from flash.core.utilities.imports import _FIFTYONE_AVAILABLE
+from flash.core.utilities.imports import _FIFTYONE_AVAILABLE, lazy_import
 
+Classification, Classifications = None, None
 if _FIFTYONE_AVAILABLE:
-    from fiftyone.core.labels import Classification, Classifications
+    fol = lazy_import("fiftyone.core.labels")
+    if TYPE_CHECKING:
```

Comment:
This may cause some issues for sphinx docs build, but I'm not sure. Seems to be working fine for now.

Reply:
I didn't realize this. Did you figure out a way around it when it came up in the past?

Reply:
Damn, I thought I remembered something but that was for circular imports so doesn't apply here 😆 I _think_ (with low confidence) that if you annotate types as `'fol.Classification'` or `'fol.Classifications'` (with quotes) then it should avoid the import of fiftyone in normal usage (but will still be triggered when type checking). That _should_ allow you to remove the `TYPE_CHECKING` clause.

Reply:
Adding quotes does work to prevent the import, but then when I run it through `mypy` I get:
```
error: Name "foc.SampleCollection" is not defined
```

Is there another type checking tool that you use other than `mypy`?

Reply:
Ideally it would work with mypy, I guess just stick with `TYPE_CHECKING` for now and we can just fix it if we start getting issues.

Reply:
Sounds good, let me know if any issues pop up.

Reply:
Will do :smiley: